src/Akeneo/AkeneoClient.cs

```diff
@@ -62,16 +62,32 @@ public AkeneoClient(Uri apiEndPoint, IAuthenticationClient authClient) : base(ap
 			return FilterAsync<TModel>(queryString, ct);
 		}
 
-		public async Task<PaginationResult<TModel>> FilterAsync<TModel>(string queryString, CancellationToken ct = default(CancellationToken)) where TModel : ModelBase
+        public Task<PaginationResult<TModel>> SearchAsync<TModel>(IEnumerable<Criteria> criterias, int limit = 10, CancellationToken ct = default(CancellationToken)) where TModel : ModelBase
+        {
+            var queryString = _searchBuilder.GetQueryString(criterias);
+            return FilterAsync<TModel>(queryString, limit, ct);
+        }
+
+        public async Task<PaginationResult<TModel>> FilterAsync<TModel>(string queryString, int limit = 10, CancellationToken ct = default(CancellationToken)) where TModel : ModelBase
```

Comment:
Just thinking out load here... the signature for `FilterAsync` has a string argument for `queryString`, and this overload also accepts a `limit` that is combined with the `queryString`... I think this overload is redundant, as the client could create the `&limiy=` query string herself. Thoughts?